### PR TITLE
Feature add sort param for consultants

### PR DESCRIPTION
**Pull request by mebo4b**
- Integrates the sort query param for admin consultant endpoint
- extends the response model of consultants with his agencies

### CODE REVIEW

File: src/main/java/de/caritas/cob/userservice/api/admin/service/agency/ConsultantAgencyAdminService.java [hunk 1]
```diff
@@ -71,6 +76,52 @@ public ConsultantAgencyResponseDTO findConsultantAgencies(String consultantId) {
         .build();
   }
 
+  public void appendAgenciesForConsultants(Set<ConsultantDTO> consultants) {
+    var consultantIds = consultants.stream()
+        .map(ConsultantDTO::getId)
+        .collect(Collectors.toSet());
+
+    var consultantAgencies = consultantAgencyRepository
```
Comment (adnanalicic) [inline src/main/java/de/caritas/cob/userservice/api/admin/service/agency/ConsultantAgencyAdminService.java:84]: in this case shown deleted consultants on frontend will not have information about agency.
image

File: api/useradminservice.yaml [hunk 2]
```diff
@@ -711,6 +720,24 @@ components:
         absent:
           type: boolean
 
+    Sort:
+      type: object
+      properties:
+        field:
+          type: string
+          example: 'firstName|lastName|username|email'
```
Comment (adnanalicic) [inline api/useradminservice.yaml:728]: lowercase or uppercase? https://happylife.develop.onlineberatung.net/service/useradmin/consultants/?page=1&perPage=10&order=ASC&field=firstName -> 400 error

Comment (adnanalicic) [reply api/useradminservice.yaml:728]: my current frontend version sends uppercase and it works

File: src/main/java/de/caritas/cob/userservice/api/admin/service/agency/ConsultantAgencyAdminService.java (hunk 1 above)
Comment (mebo4b) [reply src/main/java/de/caritas/cob/userservice/api/admin/service/agency/ConsultantAgencyAdminService.java:84]: Done

File: api/useradminservice.yaml (hunk 2 above)
Comment (mebo4b) [reply api/useradminservice.yaml:728]: I integrated ignorecase for the validation but you can leave it uppercase i think ;-)

**Pull request merged**